### PR TITLE
Readme translated to French

### PR DESCRIPTION
**Pull request by el00ruobuob**
This come along with monero-project/kovri#939
A single commit for a single file.
For review by @erciccione team.

### CODE REVIEW

File: i18n/fr/README.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+| clang               | 3.5 (3.6 on FreeBSD)         |          | clang       | clang            | clang (Apple)    | clang36       | llvm        |
+| CMake               | 3.5.1                        |          | cmake       | cmake            | cmake            | cmake         | cmake       |
+| gmake (BSD)         | 4.2.1                        |          |             |                  |                  | gmake         | gmake       |
+| Boost               | 1.58                         |          | boost       | libboost-all-dev | boost            | boost-libs    | boost       |
+| OpenSSL             | Always latest stable version |          | openssl     | libssl-dev       | openssl          | openssl       | openssl     |
+| Doxygen             | 1.8.6                        |    X     | doxygen     | doxygen          | doxygen          | doxygen       | doxygen     |
+| Graphviz            | 2.36                         |    X     | graphviz    | graphviz         | graphviz         | graphviz      | graphviz    |
+| Docker              | Always latest stable version |    X     | See website | See website      | See website      | See website   | See website |
```
Comment (erciccione) [inline i18n/fr/README.md]: This string is not translated

File: i18n/fr/README.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+| ------------------- | ---------------------------- |:--------:| ----------- | ---------------- | ---------------- | ------------- | ----------- |
+| git                 | 1.9.1                        |          | git         | git              | git              | git           | git         |
+| gcc                 | 4.9.2                        |          | gcc         | gcc              |                  |               |             |
+| clang               | 3.5 (3.6 on FreeBSD)         |          | clang       | clang            | clang (Apple)    | clang36       | llvm        |
+| CMake               | 3.5.1                        |          | cmake       | cmake            | cmake            | cmake         | cmake       |
+| gmake (BSD)         | 4.2.1                        |          |             |                  |                  | gmake         | gmake       |
+| Boost               | 1.58                         |          | boost       | libboost-all-dev | boost            | boost-libs    | boost       |
+| OpenSSL             | Always latest stable version |          | openssl     | libssl-dev       | openssl          | openssl       | openssl     |
```
Comment (erciccione) [inline i18n/fr/README.md]: "Always latest stable version" needs to be translated

File: i18n/fr/README.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+|-----------|--------|
+| Coverity  | [![Coverity Status](https://scan.coverity.com/projects/7621/badge.svg)](https://scan.coverity.com/projects/7621/)
+| Coveralls | [![Coveralls Status](https://coveralls.io/repos/github/monero-project/kovri/badge.svg?branch=master)](https://coveralls.io/github/monero-project/kovri?branch=master)
+| License   | [![License](https://img.shields.io/badge/license-BSD3-blue.svg)](https://opensource.org/licenses/BSD-3-Clause)
+
+## Compilation
+
+### Dépendances et environnements
+
+| Dependency          | Minimum version              | Optional | Arch Linux  | Ubuntu/Debian    | macOS (Homebrew) | FreeBSD       | OpenBSD     |
```
Comment (erciccione) [inline i18n/fr/README.md]: Please translate the english words used here

File: i18n/fr/README.md (hunk 1 above)
Comment (el00ruobuob) [reply i18n/fr/README.md]: ok

File: i18n/fr/README.md (hunk 2 above)
Comment (el00ruobuob) [reply i18n/fr/README.md]: ok

**Comment by el00ruobuob** [comment]
Corrections applied @erciccione

File: i18n/fr/README.md (hunk 3 above)
Comment (el00ruobuob) [reply i18n/fr/README.md]: ok

**Pull request merged**